trinity/common/experience.py

```diff
@@ -126,6 +126,10 @@ class Experience:
     chosen_text: Optional[str] = None  # Text of the chosen response
     rejected_text: Optional[str] = None  # Text of the rejected response
 
+    # for multi-modal data
+    multi_modal_data: Optional[Dict[str, Any]] = None
```

Comment:
Add some comments about the difference between `multi_modal_data` and `multi_modal_inputs`, and rename them if necessary.